### DISCUSSION TITLE
how to scroll into hidden icons

### DISCUSSION THREAD

**Discussion by endingwithali** [jordanbaird/Ice#149] [Q&A] [answered]
as a developer i have a few things running in my tool bar - the issue is that there are enough that some are hidden. in bartender i used to be able to scroll to see the next "row" of apps, but scroll on ice jsut closes it.

any recommendations here?

**Comment by jordanbaird** [comment, marked as answer]
You should be able to do this. Scroll left (or up) to show the hidden icons, right (or down) to hide them

**Reply by TheBreznsoiza** [reply]
I think she meant to scroll left and right to scroll through the hidden icons when there are more than the available space, on a MacBook with notch the free menubar space is a not enough to show them all
Maybe its a feature request 😊
